docs/src/pages/index.soy

```diff
@@ -108,7 +108,7 @@ description: "A collection of well known Algebraic Data Types for your utter enj
       <section class="highlight row">
         <div class="col-md-6 col-md-push-9">
           <h4 class="highlight-title">Use a collection of popular Algebraic Data Types (ADTs)</h4>
-          <p class="highlight-description">You have heard of things like Maybe and Either and heck maybe even IO, that is what Crocks provide.</p>
+          <p class="highlight-description">Have you ever heard of a Maybe, an Either, or, heck, even an IO? These are a few ADTs which Crocks brings to the table.</p>
```

Comment:
Still think the articles need to be removed.
We should talk about the Types and not instances of.

Reply:
We could be talking about the operation?